py-polars/polars/series/binary.py

```diff
@@ -19,7 +19,7 @@ class BinaryNameSpace:
     def __init__(self, series: Series):
         self._s: PySeries = series._s
 
-    def contains(self, literal: bytes) -> Series:
+    def contains(self, literal: bytes | Expr) -> Series:
```

Comment:
Can you use `IntoExpr` here? That accounts for all possible expression compatible inputs.